packages/mask/src/plugins/Predict/Azuro/PlaceBetDialog.tsx

```diff
@@ -0,0 +1,235 @@
+        gap: theme.spacing(0.5),
+    },
+    infoContainer: {
+        minHeight: 30,
+    },
+    infoTitle: {
+        fontWeight: '300',
+    },
+    actionButton: {
+        flexDirection: 'column',
+        position: 'relative',
+        marginTop: theme.spacing(1.5),
+        lineHeight: '22px',
+        justifyContent: 'center',
+        alignItems: 'center',
+        padding: '13px 0',
+        fontSize: 18,
+    },
+}))
+
+export interface PlaceBetDialogProps {
+    open: boolean
+    onClose?: () => void
+    game: AzuroGame
+    condition: Odds
+}
+
+export function PlaceBetDialog(props: PlaceBetDialogProps) {
+    const { open, onClose, game, condition } = props
+    const { t } = useI18N()
+    const { classes } = useStyles()
+    const [amount, setAmount] = useState('')
+    const [slippage, setSlippage] = useState<number>(2)
+    const [confirmModalIsOpen, openConfirmModal] = useBoolean(false)
+    const chainId = useChainId()
+    const { AZURO_LP } = useAzuroConstants()
+    const {
+        value: token,
+        error,
+        loading,
+    } = useFungibleToken(NetworkPluginID.PLUGIN_EVM, contractAddresses[chainId as ChainId]?.token)
+    console.log('amount: --------------------- ', amount)
+    console.log('token: --------------------- ', token)
+
+    const { value: balance } = useFungibleTokenBalance(NetworkPluginID.PLUGIN_EVM, token?.address)
+    const { value: actualRate, loading: actualRateLoading } = useActualRate(condition, amount)
+
+    const rawAmount = rightShift(String(amount), USDT_DECIMALS)
+    const deadline = Math.floor(Date.now() / 1000) + 2000
+    const { minRate, loading: minRateLoading } = useMinRate(slippage, actualRate ?? 0)
+    const rawMinRate = rightShift(minRate, RATE_DECIMALS)
+
+    const [transactionState, transactionCallback, resetTransactionCallback] = usePlaceBetCallback(
+        condition.conditionId,
+        rawAmount,
+        condition.outcomeId,
+        deadline,
+        rawMinRate,
+    )
+
+    const validationMessage = useMemo(() => {
+        if (isZero(amount) || !amount) return t('plugin_azuro_enter_an_amount')
+        if (isGreaterThan(rightShift(amount ?? 0, token?.decimals ?? 0), balance ?? 0))
+            return t('plugin_azuro_insufficient_amount')
+
+        return ''
+    }, [amount, balance, token?.decimals])
+
+    const handleSlippage = (event: React.MouseEvent<HTMLElement>, newSlippage: number) => setSlippage(newSlippage)
+    const event = `${game.participants[0].name} vs ${game.participants[1].name}`
+    const outcome = outcomeRegistry[condition.outcomeRegistryId](game)
+    const shareText = t(
+        isTwitter(activatedSocialNetworkUI) || isFacebook(activatedSocialNetworkUI)
+            ? 'plugin_azuro_share'
+            : 'plugin_azuro_share_no_official_account',
+        {
+            account: isTwitter(activatedSocialNetworkUI) ? t('twitter_account') : t('facebook_account'),
+        },
+    )
```

Comment:
You can use i18next's context feature.

Reference https://github.com/DimensionDev/Maskbook/pull/6566/files

Reply:
Done.